### PR TITLE
fix: add docs for starting ephemeral workspaces

### PR DESCRIPTION
**Pull request by akurinnoy**
## What does this pull request change?

This PR adds some info about the URL parameter storageType that can be used for starting a new workspace.

## What issues does this pull request fix or reference?

https://github.com/eclipse/che/issues/21569#issuecomment-1198060211

## Specify the version of the product this pull request applies to

7.55

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/end-user-guide/pages/url-parameter-for-starting-ephemeral-workspaces.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+The URL parameter `__<storage_type>__` values for supported storage types are `ephemeral`, `per-user` and `per-workspace`.
+
+The URL for starting an ephemeral workspace:
+
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__<git_repository_url>__?storageType=ephemeral
+----
+
+If the URL for starting a new workspace doesn't contain a URL parameter specifying the storage type, {prod-short} uses the default storage type defined in the `CheCluster` Custom Resource.
```
Comment (ibuziuk) [inline modules/end-user-guide/pages/url-parameter-for-starting-ephemeral-workspaces.adoc]:
```suggestion
If the URL for starting a new workspace does not contain a URL parameter specifying the storage type, {prod-short} uses the default storage type defined in the `CheCluster` Custom Resource.
```
will let @max-cx @themr0c review but as I recall we do not use contractions in the official docs

File: modules/end-user-guide/pages/url-parameter-for-starting-ephemeral-workspaces.adoc [hunk 2]
```diff
@@ -0,0 +1,30 @@
+:_content-type: CONCEPT
+:description: URL parameter starting ephemeral workspaces
```
Comment (ibuziuk) [inline modules/end-user-guide/pages/url-parameter-for-starting-ephemeral-workspaces.adoc]:
```suggestion
:description: URL parameter for starting ephemeral workspaces
```

**Review by ibuziuk — APPROVED** [review]
LGTM, added two suggestions

**Review by max-cx — APPROVED** [review]
LGTM

**Pull request merged**